modules/loki-rbac-rules-permissions.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * logging/logging_alerts/custom-logging-alerts.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="loki-rbac-rules-permissions_{context}"]
+= Authorizing LokiStack rules RBAC permissions
+
+Administrators can allow users to create and manage their own alerting and recording rules by binding cluster roles to usernames.
+Cluster roles are defined as `ClusterRole` objects that contain necessary role-based access control (RBAC) permissions for users.
+
+In logging 5.8 and later, the following cluster roles for alerting and recording rules are available for LokiStack:
+
+[options="header"]
+|===
+|Rule name |Description
+
+|`alertingrules.loki.grafana.com-v1-admin`
+|Users with this role have administrative-level access to manage alerting rules. This cluster role grants permissions to create, read, update, delete, list, and watch `AlertingRule` resources within the `loki.grafana.com/v1` API group.
+
+|`alertingrules.loki.grafana.com-v1-crdview`
+|Users with this role can view the definitions of Custom Resource Definitions (CRDs) related to `AlertingRule` resources within the `loki.grafana.com/v1` API group, but do not have permissions for modifying or managing these resources.
+
+|`alertingrules.loki.grafana.com-v1-edit`
+|Users with this role have permission to create, update, and delete `AlertingRule` resources.
```

Comment:
Do we need to mention that Users having 'edit' role must also have 'view' role in order to edit an AlertingRule? cc. @xperimental

Reply:
I'd say someone (as in "a person") who's allowed to edit items usually also has the right to view it, so I would say _people_ mostly apply either "view" or "admin" (which includes both view and edit).

Technically it is not true though, that you need view to be able to update a rule. If you have the "edit" role then you can do the update API call needed for the update. The question is then just "how do I get the current value?", for which tools like `oc edit` usually get the current server state (and thus need "view").

If that is not needed, for example because it's not a person doing this, but a Jenkins job that just takes the current value from a repository and applies it and does not care what the "server version" is, then you're fine just having "edit".